wiki/Contributing-to-Oppia-Android.md

```diff
@@ -0,0 +1,103 @@
+_These instructions are for developers who'd like to contribute code to improve the Oppia platform. If you'd prefer to help out with other things, please see our [[general contribution guidelines|Home]]._
+
+Thanks for your interest in contributing to the Oppia Android project, and making it easier for students to learn online in an effective and enjoyable way!
+
+If you run into any problems along the way, we're here to help! Check out our [[wiki page on getting help|Get-Help]] for the communication channels you can use. If you find any bugs, you can also file an issue on our [issue tracker](https://github.com/oppia/oppia-android/issues). There are also lots of helpful resources in the sidebar, check that out too!
```

Comment:
Ditto in this file--the ``[[`` doesn't seem to be rendering correctly.

Reply:
*Ditto